source/DSCResources/MSFT_SccmIniFIle/MSFT_SccmIniFIle.psm1

```diff
@@ -0,0 +1,1107 @@
+$script:configMgrResourcehelper = Join-Path -Path $PSScriptRoot -ChildPath '..\..\Modules\ConfigMgrCBDsc.ResourceHelper'
+
```

Comment:
This is a module and should contain a LocalizedData file  and all Write-Verbose messages should be changed over to use it

Reply:
With the new build they created a dsccommon module that gets added in the build. I added that to this. I will update the verbose messages to use the file.